src/App.scss

```diff
@@ -1,4 +1,17 @@
 .App {
```

Comment:
Please don't use `scss` files here, as you have Material-UI library and all the classes should be declared inside of it. It's redundant here to have so many style libraries.

Reply:
done